bindings/nodejs/client_context.hpp

```diff
@@ -29,10 +29,10 @@ class ClientContext
 public:
     using EventCallback = std::function<void()>;
     static std::unique_ptr<ClientContext>
-    create(const std::string& source, const std::string& map_files, EventCallback event_callback);
+    create(const std::string& source, const std::string& map_files, const bool wait_for_server, EventCallback event_callback);
```

Comment:
I don't see why this should accept `wait_for_server` from the caller. It should be always false for nodejs bindings, so you can just use false here.

Reply:
Done